### ISSUE TITLE
DAP accessibility violations for cv-toast-notification

### ISSUE THREAD

**Issue by sabinead** [carbon-design-system/carbon-components-vue#435]
DAP accessibility tool shows the following violations for cv-toast-notification:

- Each form control must have associated label.
   -> close button has no aria-label

- The attribute identifier referenced by the element identifier is not a valid WAI-ARIA State or Property.
   The attribute aria-role referenced by the element DIV is not a valid WAI-ARIA State or Property.
   -> should be role="alert"
       <div data-v-4bb7277b="" data-notification="" **aria-role="alert"** class="cv-notifiation bx--toast-notification bx--toast-notification--error">...</div>
